### PR TITLE
Update com.google.Chrome.plist

### PR DESCRIPTION
**Pull request by BigMacAdmin**
Adding 4 new "privacy sandbox ad controls" for Google Chrome

https://chromeenterprise.google/policies/#PrivacySandbox

### CODE REVIEW

**Comment by relgit** [comment]
Thank for the submission; the changes look good to me. Now we just need to have `pfm_version` bumped, `pfm_last_modified` updated, the new keys added to `pfm_segments`, and we're good to go.

**Comment by BigMacAdmin** [comment]
@relgit 
Is the pfm_segments array the deciding factor that determines which tab a configuration key is displayed under?
If so, and additionally: I've added 4 new keys for the Google Chrome profile related to ad privacy. Which tab would these be appropriate for? Misc. or maybe Content Settings? Or does this justify an entirely new tab for "Privacy" or "Ad Tracking"?

**Comment by relgit** [comment]
Yes, `pfm_segments` determines that tab that subkeys appear under.

I can see that Google chose to group these new four subkey on their own and not as part of the Misc. group, and honestly Misc. is already very crowded, so yes, lets create a new _Privacy Sandbox_ group. To create a new group you'll need to also amend the `PFC_SegmentedControl_0` subkey.

**Comment by BigMacAdmin** [comment]
There must be something I'm missing, I've followed your instructions but I cannot seem to get a new "Privacy Sandbox" group created.

The new keys still show in the "Other" section within iMazing Profile Editor.

File: Manifests/ManagedPreferencesApplications/com.google.Chrome.plist [hunk 1]
```diff
@@ -15,7 +15,7 @@
 	<key>pfm_interaction</key>
 	<string>combined</string>
 	<key>pfm_last_modified</key>
-	<date>2023-08-15T08:00:00Z</date>
+  <date>2023-10-05T00:00:00Z</date>
```
Comment (relgit) [inline Manifests/ManagedPreferencesApplications/com.google.Chrome.plist:18]: Line indentation shrunk.

File: Manifests/ManagedPreferencesApplications/com.google.Chrome.plist [hunk 2]
```diff
@@ -9753,6 +9849,5 @@ If the policy DeveloperToolsAvailability is set, the value of the policy Develop
 	<key>pfm_unique</key>
 	<true/>
 	<key>pfm_version</key>
-	<integer>14</integer>
-</dict>
```
Comment (relgit) [inline Manifests/ManagedPreferencesApplications/com.google.Chrome.plist:9757]: Dictionary closing tag lost.

**Review by relgit — CHANGES_REQUESTED** [review]
That's OK. I'm assuming this was tested on iMazing Profile Editor, which has a grouping override on the Chrome manifest. To show the actual grouping that's in the manifest, you can untick _Process custom manifests for optimal display in iMazing Profile Editor (recommended)_ in the app settings.
The override will be updated in the next release to fall in line with this pull request.

Now, the grouping changes look good, but have a look please at my comments for two small issues that require addressing.

Thanks!

File: Manifests/ManagedPreferencesApplications/com.google.Chrome.plist (hunk 1 above)
Comment (BigMacAdmin) [reply Manifests/ManagedPreferencesApplications/com.google.Chrome.plist]: Fixed, sorry not sure how this happened.

File: Manifests/ManagedPreferencesApplications/com.google.Chrome.plist (hunk 2 above)
Comment (BigMacAdmin) [reply Manifests/ManagedPreferencesApplications/com.google.Chrome.plist:9757]: Fixed, sorry not sure how this happened.

**Review by relgit — APPROVED** [review]

**Pull request merged**